queries/liquidityPools/useYearnSNXVaultQuery.ts

```diff
@@ -79,15 +78,15 @@ const useYearnSNXVaultQuery = (options?: QueryConfig<YearnVaultData>) => {
 
 			return {
 				address: yearnSNXVault.address,
-				balance,
 				userBalance: userBalance,
 				userBalanceBN: toBigNumber(userBalance),
 				distribution: 0,
 				duration: 0,
 				periodFinish: Date.now() * 2, // never expires
 				rewards: 0,
-				staked: staked.toNumber(),
-				stakedBN: staked,
+				staked: stakedBN.toNumber(),
+				stakedBN: stakedBN,
+				stakedSNX: stakedBN.times(pricePerShare),
```

Comment:
what is `stakedSNX: stakedBN.times(pricePerShare)` exactly? I would have thought `staked` was already giving you the staked SNX balance